dll/win32/shell32/brfolder.cpp

```diff
@@ -613,18 +615,12 @@ BrFolder_OnInitDialog(HWND hWnd, BrFolder *info)
 
         if (lpBrowseInfo->ulFlags & BIF_NEWDIALOGSTYLE)
             info->layout = LayoutInit(hWnd, g_layout_info, _countof(g_layout_info));
-        else
```

Comment:
What's the value in `info->layout` in case the if-block is not taken? Is it left uninitialized? If so, it would be worth either keeping these original `info->layout = NULL;` there, or ensuring it's initialized to NULL (and stays like that) somewhere else
(for example, putting the `info->layout = NULL;` just before the `if (lpBrowseInfo->ulFlags & BIF_USENEWUI)` above, line 596)

Reply:
Set `NULL` to `info->layout`.